tests/leaderelection_test.go

```diff
@@ -133,57 +133,69 @@ var _ = Describe("[rfe_id:1250][crit:high][vendor:cnv-qe@redhat.com][level:compo
 
 var _ = Describe("[rfe_id:1250][crit:high][test_id:1889][vendor:cnv-qe@redhat.com][level:component]Leader election tests during import", func() {
 	f := framework.NewFrameworkOrDie("leaderelection-test")
+	var (
+		leaderPodName string
+		newDeployment *appsv1.Deployment
+		err           error
+	)
 
 	BeforeEach(func() {
-		By("Unsubscribing from OLM, so we can stop the operator")
-		// TODO once we have olm integration completed.
+		By("Check only one CDI controller deployment/pod")
```

Comment:
I like the beforeeach/aftereach for this test.  Does it make sense to do the same for the other test?  Maybe have some shared setup/teardown functions?  If not, it's fine.

Reply:
it probably does make sense for a future us, so we aren't copying and pasting the same code over and over and over again. Let me fix it up so our code is better.